x/pool-incentives/client/cli/tx.go

```diff
@@ -9,30 +9,13 @@ import (
 
 	"github.com/cosmos/cosmos-sdk/client"
 	sdk "github.com/cosmos/cosmos-sdk/types"
-	"github.com/cosmos/cosmos-sdk/x/gov/client/cli"
+	govcli "github.com/cosmos/cosmos-sdk/x/gov/client/cli"
 	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
 
 	"github.com/osmosis-labs/osmosis/v7/osmoutils"
 	"github.com/osmosis-labs/osmosis/v7/x/pool-incentives/types"
 )
 
-func NewTxCmd() *cobra.Command {
```

Comment:
hrmm is it safe to delete this / is there a reason behind deleting this?

Reply:
It is decided in #1888 that submit proposal cmd should only have gov prefix, so these command will be registered with gov handler instead of tx cmd of incentives module.

Reply:
> LGTM - I think we should create a changelog entry for this

Should I add this to state breaking entry or not

Reply:
imo should go under breaking changes